system/Debug/Toolbar/toolbarloader.js.php

```diff
@@ -1,17 +1,27 @@
 
```

Comment:
**TODO:** Detect AJAX Requests and show it on the History or auto get latest updates, updating the HTML on the fly.